src/intellisense/langIntellisenseProvider.ts

```diff
@@ -394,7 +394,10 @@ class CommandCompletionProvider extends IntellisenseProvider implements vscode.C
         const wordRange = document.getWordRangeAtPosition(position, this.contextRegex);
         if (wordRange) {
             const partial = document.getText(wordRange);
-            const delRange = new vscode.Range(wordRange.start, wordRange.start.translate(0, 1));
+            // get pos of \ from partial
+            const match = partial.search(/\\/);
```

Comment:
make sure this is correct by yourself.

Reply:
In case of unknown error, the position is now determined by regex search.